Firmware/ConfigurationStore.cpp

```diff
@@ -32,7 +32,7 @@ static bool EEPROM_writeData(uint8_t* pos, uint8_t* value, uint8_t size, const c
 #endif //DEBUG_EEPROM_WRITE
 {
 #ifdef DEBUG_EEPROM_WRITE
-	printf_P(PSTR("EEPROM_WRITE_VAR addr=0x%04x size=0x%02hhx name=%s\n"), pos, size, name);
+	printf_P(PSTR("EEPROM_WRITE_VAR addr=0x%04x size=0x%02x name=%s\n"), pos, size, name);
```

Comment:
please use formatting flags defines from `inttypes.h`
https://manned.org/PRIu8

Reply:
In variadic functions, uint8_t always gets promoted to 16bit. hh/h on avr doesn’t do anything for printf. It only does stuff for scanf.

Reply:
Reference: https://www.nongnu.org/avr-libc/user-manual/group__avr__stdio.html#:~:text=The%20hh%20length%20modifier%20is%20ignored%20(char%20argument%20is%20promouted%20to%20int).%20More%20exactly%2C%20this%20realization%20does%20not%20check%20the%20number%20of%20h%20symbols.